tests/external/iree-test-suites/onnx_ops/onnx_ops_cpu_llvm_sync.json

```diff
@@ -410,11 +342,13 @@
     "onnx/node/generated/test_qlinearmatmul_3D_uint8_float16",
     "onnx/node/generated/test_qlinearmatmul_3D_uint8_float32",
     "onnx/node/generated/test_reduce_max_bool_inputs",
-    "onnx/node/generated/test_reduce_min_empty_set",
+    "onnx/node/generated/test_reduce_max_empty_set",
     "onnx/node/generated/test_reduce_min_bool_inputs",
+    "onnx/node/generated/test_reduce_min_empty_set",
     "onnx/node/generated/test_reduce_sum_empty_set_non_reduced_axis_zero",
     "onnx/node/generated/test_resize_downsample_scales_cubic_align_corners",
     "onnx/node/generated/test_resize_downsample_scales_linear_align_corners",
+    "onnx/node/generated/test_scan_sum",
```

Comment:
This test was passing but the importer changed its function signatures for some reason: https://github.com/iree-org/iree-test-suites/pull/42#discussion_r1850839336

```diff
- --input=2xf32=@input_0.bin
- --input=3x2xf32=@input_1.bin
- --expected_output=2xf32=@output_0.bin
- --expected_output=3x2xf32=@output_1.bin
+ --input=1x2xf32=@input_0.bin
+ --input=1x3x2xf32=@input_1.bin
+ --expected_output=1x2xf32=@output_0.bin
+ --expected_output=1x3x2xf32=@output_1.bin
```

Possibly a bug in the older ONNX version? Prior to that PR, input and output types were loaded from the _upgraded onnx model_. With that PR, input and output types are loaded from the _original onnx model_.